SharpMUSH.MarkupString/Markup/Markup.fs

```diff
@@ -106,6 +106,10 @@ module MarkupImplementation =
                 match acc.IndexOf("\u001b[", currentIndex, System.StringComparison.Ordinal) with
                 | -1 -> acc
                 | escapeCodeStartIndex ->
+                    // TODO: Implement a case that turns:
```

Comment:
@coderabbitai Help me figure this out.

Reply:
Yes, please propose multiple specific implementations.

Reply:
Let's focus on the regex version. We can generalize a lot more to the full:
`\u001b[` 'until' `m` pattern. So in regex: (\u001b[^m]*m) as a capture group. Improve the regex.